src/Fable.ReactGoogleMaps/README.md

```diff
@@ -134,3 +134,33 @@ let myMap =
 let bounds = mapRef.GetBounds()
 
 ```
+
+### Set bounds of map to fit all the markers
+
+```fs
+    // ...
+    let markerPositions: Position list = // ...
+        
+    // In some scenarios ref can be null, f.ex. slow network when the google maps isn't fully loaded yet.
+    let setRef (ref:obj) =
+        let bounds = ReactGoogleMaps.Coordinates.newLatLngBounds()
+        
+        match Option.ofObj ref with
```

Comment:
Just if isNull ref then would be simpler here

Reply:
Didn't know that one, but maybe the definition of `SetRef` should be `| SetRef of (obj option -> unit)`. That way it is explicit.

Reply:
well let me move the nullCheck into MapRef itself